cmd/root.go

```diff
@@ -11,6 +11,7 @@ package cmd
 
 import (
 	"os"
+	"reanahub/reana-client-go/pkg/templater"
```

Comment:
Hmm, `templater` seems rather generic, as it could refer to our own outputs. What about naming the package like `cobratemplates` or `cobracommands` or `cobragroups` or `cligroups` or `commandgroups` or something of that kind?

Reply:
thanks, I like `commandgroups` the most